3-networks-dual-svpc/modules/restricted_shared_vpc/service_control.tf

```diff
@@ -66,14 +66,22 @@ module "regular_service_perimeter" {
   policy         = var.access_context_manager_policy_id
   perimeter_name = local.perimeter_name
   description    = "Default VPC Service Controls perimeter"
-  resources      = [var.project_number]
-  access_levels  = [module.access_level_members.name]
 
-  restricted_services     = var.restricted_services
-  vpc_accessible_services = ["RESTRICTED-SERVICES"]
+  # configurations for a perimeter in dry run mode.
+  resources_dry_run               = [var.project_number]
```

Comment:
the way the projects are added to the perimeter in step 4-projects also needs to be updated.

```
  vpc_service_control_attach_enabled = var.vpc_service_control_attach_enabled
  vpc_service_control_perimeter_name = var.vpc_service_control_perimeter_name
  vpc_service_control_sleep_duration = var.vpc_service_control_sleep_duration
```  
https://github.com/terraform-google-modules/terraform-example-foundation/blob/master/4-projects/modules/single_project/main.tf#L62C1-L64C78

This cloud be a new flag like how it is coded in the project factory module

https://github.com/terraform-google-modules/terraform-google-project-factory/blob/master/modules/core_project_factory/main.tf#L347C1-L365C2

Reply:
hey Eliot, just wondering if this change could be done with a flag instead of comment/uncomment the code, export in outputs to be read by other steps, like 4-projects as Daniel mentioned. It would also be more transparent for others blueprints that have documentation of how to deploy on top of foundation.